lib/stormpath-sdk/provider/github/github_provider.rb

```diff
@@ -14,5 +14,5 @@
 # limitations under the License.
 #
 class Stormpath::Provider::GithubProvider < Stormpath::Provider::Provider
-  prop_reader :client_id, :client_secret
+  prop_reader :client_id, :client_secret, :user_info_mapping_rules
```

Comment:
`has_many :user_info_mapping_rules`

This should go to the base class: `Provider`.